free-1.md

```diff
@@ -57,12 +106,23 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I would say 90% of the time would be working on the system to parse calendar
+data and design the interface for the app. 
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I would say this project is decently ambitious as it wasn't defined what type of 
+calendars it is able to sync with or what phones it would be compatable with. 
+So it could take dozens of calendar types and support all 3 types of phone OSs. 
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This would be a good idea since it wouldn't be too hard to find users to test on
+and probably a lot that would be interested in using it. It would be a bad idea
+because of the scope and the fact that there are already so many ringtone time 
+setters out there on the market. 
+
```

Comment:
I think this is a really interesting and unusual idea. I'm not sure about scope--it may be a bit small--but I'd be willing to go forward with this project.

As you mentioned, the project can't be about designing the app. You'd have to design the language (e.g., probably the intermediate representation) first, then worry about the interface. But with those restrictions, it's possible that you wouldn't enjoy the project as much.